lib/constants.js

```diff
@@ -53,5 +53,7 @@ module.exports = {
     AMQP_DEFAULT_QUEUE: 'iotaqueue',
     AMQP_DEFAULT_DURABLE: true,
     AMQP_DEFAULT_RETRIES: 5,
-    AMQP_DEFAULT_RETRY_TIME: 5
+    AMQP_DEFAULT_RETRY_TIME: 5,
+
+    X_PROCESSING_TIME: 'X-Processing-Time'
```

Comment:
It is based in some existing (pseudo)standard? Or it has been fully invented?

Reply:
It just a new invented header. But we can use other...

Reply:
Maybe we should do a little searching to check is there is something similar we can reuse or inspire on...

Reply:
Maybe X-Process-Time matches better